File: service/SurfWebAutofac/Model/Caches/MapWrCache.cs
using Model.Models.Main;

namespace Model.Caches;

public class MapWrCache
{
    /// <summary>
    ///     地图id
    /// </summary>
    public long MapId { get; set; }

    /// <summary>
    ///     地图名称
    /// </summary>
    public string? MapName { get; set; }

    /// <summary>
    ///     难度
    /// </summary>
    public string Difficulty { get; set; }

    /// <summary>
    ///     地图图片
    /// </summary>
    public string Img { get; set; }

    /// <summary>
    ///     玩家id
    /// </summary>
    public long PlayerId { get; set; }

    /// <summary>
    ///     玩家名
    /// </summary>
    public string PlayerName { get; set; }

    /// <summary>
    ///     时间
    /// </summary>
    public float Time { get; set; }

    /// <summary>
    ///     日期
    /// </summary>
    public DateTime Date { get; set; }

    /// <summary>
    ///     通关类型(0-主线，1-奖励，2-阶段)
    /// </summary>
    public RecordTypeEnum Type { get; set; }

    /// <summary>
    ///     阶段
    /// </summary>
    public int? Stage { get; set; }
}

File: service/Java/SurfWeb/SurfWeb/system/src/main/java/com/surfweb/system/caches/MapWrCache.java
package com.surfweb.system.caches;


import com.surfweb.common.enums.RecordTypeEnum;
import lombok.Data;

import java.util.Date;

/**
 * 地图WR缓存
 */
@Data
public class MapWrCache {

    /**
     * 地图id
     */
    private long mapId;

    /**
     * 地图名称
     */
    private String mapName;

    /**
     * 难度
     */
    private String difficulty;

    /**
     * 地图图片
     */
    private String img;

    /**
     * 玩家id
     */
    private long playerId;

    /**
     * 玩家名
     */
    private String playerName;
    /**
     * 时间
     */
    private float time;
    /**
     * 日期
     */
    private Date date;
    /**
     * 通关类型(0-主线，1-奖励，2-阶段)
     */
    private RecordTypeEnum type;
    /**
     * 阶段
     */
    private Integer stage;

}